integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/TestUtils.java

```diff
@@ -239,6 +239,58 @@ public static boolean callWebAppAndWaitTillReady(String curlCmd, int maxIteratio
     return false;
   }
 
+  /**
+   * Call a web app and wait for the specified HTTP status code.
+   * @param curlCmd curl command to call the web app
+   * @param maxIterations max iterations to call the curl command
+   * @param httpStatusCode HTTP status code
+   * @return true if specified HTTP status code is returned, false otherwise
+   */
+  public static boolean callWebAppAndWaitTillReturnedCode(String curlCmd, String httpStatusCode, int maxIterations)  {
```

Comment:
can you refactor the other callWebAppAndWaitTillReady method in this class to call callWebAppAndWaitTillReturnedCode method? Otherwise, it looks like lot of duplicated code in these two methods.

Reply:
Sure.